modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl

```diff
@@ -104,15 +105,15 @@
 </#if>
```

Comment:
Hey, Bruno! You need to use the same approach that we used in the Apps Search Results component